### PR TITLE
Ports Lawnmowers

### PR DESCRIPTION
**Pull request by AliiTea**
Ports Hippie lawnmowers

:cl: Nexendia
add: Added Hippie Lawnmowers
add: They can be purchased by traitor botanists.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
needs `to_chat()`

File: hippiestation/code/modules/vehicles/pimpin_ride.dm [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	var/driver
+
+/obj/vehicle/lawnmower/emagged
+	emagged = TRUE
+
+/obj/vehicle/lawnmower/emag_act(mob/user)
+	if(emagged)
+		user << "<span class='warning'>The safety mechanisms on \the [src] are already disabled!</span>"
+		return
+	user << "<span class='warning'>You disable the safety mechanisms on \the [src].</span>"
```
Comment (tortellinitony) [inline hippiestation/code/modules/vehicles/pimpin_ride.dm]: to chat

File: hippiestation/code/modules/vehicles/pimpin_ride.dm [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	var/list/drive_sounds = list('hippiestation/sound/effects/mowermove1.ogg', 'hippiestation/sound/effects/mowermove2.ogg')
+	var/list/gib_sounds = list('hippiestation/sound/effects/mowermovesquish.ogg')
+	var/driver
+
+/obj/vehicle/lawnmower/emagged
+	emagged = TRUE
+
+/obj/vehicle/lawnmower/emag_act(mob/user)
+	if(emagged)
+		user << "<span class='warning'>The safety mechanisms on \the [src] are already disabled!</span>"
```
Comment (tortellinitony) [inline hippiestation/code/modules/vehicles/pimpin_ride.dm]: to chat

**Review by tortellinitony — CHANGES_REQUESTED** [review]

**Comment by AliiTea** [comment]
@tortellinitony done

**Review by tortellinitony — APPROVED** [review]

**Pull request merged**